### ISSUE TITLE
Comments lose their highlighting

### ISSUE THREAD

**Issue by yaronelh** [mark-wiemer/ahkpp#362]
## Code snippet

```
; When entering a :: Inside of a comment line it's losing its comment highlighting 

; another thing
 ; When adding a space before to start up a line the commented line is not highlighted as a comment
; You can copy and paste this section of code to see it in action

```

## Expected behavior

The two scenarios should be treated as a comment

## Editor tokens and scopes

none

## Additional context

none

**Comment by fade2gray** [comment]
No problem here, maybe you have some other conflicting extension installed?

![AHK++](https://github.com/mark-wiemer/ahkpp/assets/1332009/1f599191-b41d-4253-8953-077ad5ab8092)

**Comment by mark-wiemer** [comment]
@yaronelh are you sure you're using the latest version of AutoHotkey Plus Plus?

**Comment by yaronelh** [comment]
You were correct, it seems to have been another extension although it was disabled. Basically I completely uninstalled it and then restarted a VS code.
After the restart it was gone

**Comment by kyklish** [comment]
It was fixed in #358, but it's not released yet.
